Sample/SampleForageSDK/Sections/CardNumber/CardNumberView.swift

```diff
@@ -371,19 +450,23 @@ class CardNumberView: UIView {
     }
     
     private func updateState(state: ObservableState) {
-        if (state.isEmpty) {
-            statusLabel.text = "Enter Card Number"
-            statusLabel.textColor = .gray
-        } else if (state.isComplete && state.isValid) {
-            statusLabel.text = "Card Number Valid"
-            statusLabel.textColor = .green
-        } else if (state.isValid && !state.isComplete) {
-            statusLabel.text = "Identifying Card Number"
-            statusLabel.textColor = .gray
-        } else {
-            statusLabel.text = "Card Number Invalid"
-            statusLabel.textColor = .red
-        }
+        firstResponderLabel.text = "isFirstResponder: \(state.isFirstResponder)"
+        completeLabel.text = "isComplete: \(state.isComplete)"
+        emptyLabel.text = "isEmpty: \(state.isEmpty)"
+        validLabel.text = "isValid: \(state.isValid)"
+//        if (state.isEmpty) {
+//            statusLabel.text = "Enter Card Number"
+//            statusLabel.textColor = .gray
+//        } else if (state.isComplete && state.isValid) {
+//            statusLabel.text = "Card Number Valid"
+//            statusLabel.textColor = .green
+//        } else if (state.isValid && !state.isComplete) {
+//            statusLabel.text = "Identifying Card Number"
+//            statusLabel.textColor = .gray
+//        } else {
+//            statusLabel.text = "Card Number Invalid"
+//            statusLabel.textColor = .red
+//        }
```

Comment:
can we remove this pls?

Reply:
[1b308b2](https://github.com/teamforage/forage-ios-sdk/pull/89/commits/1b308b2e3871d7b9864b3b4b61a85ef3d5157b67)